addons/mail/static/src/models/thread_cache/thread_cache.js

```diff
@@ -52,10 +53,13 @@ function factory(dependencies) {
                 return;
             }
             const messageIds = this.fetchedMessages.map(message => message.id);
-            const fetchedMessages = this._loadMessages({
-                extraDomain: [['id', '>', Math.max(...messageIds)]],
```

Comment:
I would just add `, 0` in `Math.max()` as a fix, so that fix is very minimal.

Reply:
why are you going to add a domain that does not contribute anything